Package.swift

```diff
@@ -122,6 +122,10 @@ let package = Package(
                         .product(name: "SnapshotTesting", package: "swift-snapshot-testing")
                     ],
                     exclude: ["Templates/__Snapshots__", "Data/__Snapshots__", "TestPlans"],
-                    resources: [.copy("Resources/header.heic"), .copy("Resources/background.heic")])
+                    resources: [
+                        .copy("Resources/header.heic"),
+                            .copy("Resources/background.heic"),
+                            .copy("PaywallsV2/__PreviewResources__")
```

Comment:
Huge nit: indentation
```suggestion
                        .copy("Resources/header.heic"),
                        .copy("Resources/background.heic"),
                        .copy("PaywallsV2/__PreviewResources__")
```